accord-core/src/test/java/accord/utils/MessageTask.java

```diff
@@ -119,7 +118,7 @@ public void onSuccess(Node.Id from, Reply reply)
             Invariants.checkArgument(reply == SUCCESS || reply == FAILURE);
             if (reply == FAILURE)
             {
-                originator.send(from, request, commandStore, this);
+                originator.send(from, request, MoreExecutors.directExecutor(), this);
```

Comment:
I don't 100% follow why some of the time we are OK with direct executor, but other times we want to pick a random command store.

If we have things that don't neatly fit in a command store would it make sense to have a small thread pool just for those?

One of the things I was thinking about was observability and how that could tie into the statistics we expose for thread pools and stages. We currently don't expose Accord as a stage.

Maybe we will head in a different direction, but it seems nice to be able to pull both Accord and other stages stats from the same place.

Reply:
This is a testing class that is used for burn test to change epoch over the run.  I could change this to run in the new global executor that the burn test creates, but we are already running there, so went with direct just to simplify the diff.


I pushed a patch that uses the burn test executor, and in the tracker test to use the same delay simulation executor